recipes/el-secretario-keybindings

```diff
@@ -0,0 +1,4 @@
+(el-secretario-keybindings
+ :fetcher git
+ :url "https://git.sr.ht/~zetagon/el-secretario"
+ :files ("el-secretario-keybindings.el"))
```

Comment:
`el-secretario-keybindings.el` with package-lint 20211122.1152:
```
5 issues found:
42:1: warning: `eval-after-load' is for use in configurations, and should rarely be used in packages.
48:1: warning: `eval-after-load' is for use in configurations, and should rarely be used in packages.
54:1: warning: `eval-after-load' is for use in configurations, and should rarely be used in packages.
65:1: warning: `eval-after-load' is for use in configurations, and should rarely be used in packages.
91:1: warning: `eval-after-load' is for use in configurations, and should rarely be used in packages.
```

Reply:
This is the main reason why I was hesitant to put this part as its own package. I need those `eval-after-load`s there because they define default keybindings for the rest of the packages. I must have missed something before though, because I just found how to do a thing I wanted to do without `general.el`. I will probably remove this recipe from this PR.